src/api/springApi.js

```diff
@@ -3,7 +3,7 @@ import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
 export const springApi = createApi({
   reducerPath: 'springApi',
   baseQuery: fetchBaseQuery({
-    baseUrl: 'http://localhost:5000/'
+    baseUrl: 'https://spring-react-server.herokuapp.com'
```

Comment:
And now you can't develop locally. You should support both approaches. Hint: env vars.